tests/testthat/test-read-output.R

```diff
@@ -188,11 +192,16 @@ for (.tc in .test_cases) {
 for (.tc in .test_cases) {
   test_that(.tc[[".test_name"]], {
     df <- check_ext(.tc[[".test_arg"]], .iter_floor = NULL)
-    ref_df <- readRDS(EXT_REF_FLOOR_NULL)
-    expect_identical(df, ref_df)
+    ref_df <- dget(EXT_REF_FLOOR_NULL)
+
+    # This mutate call is because of bug in testthat::expect_equal
+    # we can remove this when we switch to testthat 3e
+    df    <-      df %>% mutate(OBJ = round(OBJ, 5))
+    ref_df <- ref_df %>% mutate(OBJ = round(OBJ, 5))
+    expect_equal(df, ref_df)
 
-    ref_df <- readRDS(EXT_REF_FLOOR_0)
-    expect_failure(expect_identical(df, ref_df))
+    ref_df <- dget(EXT_REF_FLOOR_0)
+    expect_failure(expect_equal(df, ref_df))
```

Comment:
Why change from `expect_identical()` to `expect_equal()`?

Reply:
because of the rounding/digits-of-precision on the floats